src/test/java/com/alibaba/easyexcel/test/util/StringUtilsTest.java

```diff
@@ -0,0 +1,796 @@
+    Assert.assertFalse(StringUtils.hasText("\n\n"));
+  }
+
+  @Test
+  public void hasTextInputNotNullOutputTrue() {
+    Assert.assertTrue(StringUtils.hasText("foo"));
+  }
+
+  @Test
+  public void hasTextInputNotNullOutputTrue2() {
+    Assert.assertTrue(StringUtils.hasText("\n\u0000\u0000??"));
+  }
+
+  @Test
+  public void hasTextInputNotNullOutputTrue3() {
+    final CharSequence str =
+        "\n\u0000\u0000?????????????????????????????????????????????????????????????";
+    Assert.assertTrue(StringUtils.hasText(str));
+  }
+
+  @Test
+  public void hasTextInputNullOutputFalse() {
+    Assert.assertFalse(StringUtils.hasText(null));
+  }
+
+  @Test
+  public void hasTextInputNullOutputFalse2() {
+    Assert.assertFalse(StringUtils.hasText(null));
+  }
+
+  @Test
+  public void mergeStringArraysInput00Output0() {
+  	Assert.assertArrayEquals(new String[] {},
+			  StringUtils.mergeStringArrays(new String[]{}, new String[]{}));
+  }
+
+  @Test
+  public void mergeStringArraysInput1NullOutput1() {
+  	Assert.assertArrayEquals(new String[] {null},
+			  StringUtils.mergeStringArrays(new String[]{null}, null));
+  }
+
+  @Test
+  public void mergeStringArraysInput12Output2() {
+  	Assert.assertArrayEquals(new String[] {"", null},
+			StringUtils.mergeStringArrays(new String[]{""}, new String[]{null, null}));
+  }
+
+  @Test
+  public void parseTimeZoneStringInputNotNullOutputIllegalArgumentException() {
+  	thrown.expect(IllegalArgumentException.class);
+    StringUtils.parseTimeZoneString("a\'b\'c");
+  }
+
+  @Test
+  public void parseTimeZoneStringInputNotNullOutputNotNull() {
+  	Assert.assertEquals("America/Los_Angeles",
+			((ZoneInfo) StringUtils.parseTimeZoneString("America/Los_Angeles")).getID());
+    Assert.assertEquals(-28800000,
+			((ZoneInfo) StringUtils.parseTimeZoneString("America/Los_Angeles")).getRawOffset());
+  }
+
+  @Test
+  public void pathEqualsInputNotNullNullOutputFalse() {
+    Assert.assertFalse(StringUtils.pathEquals("", null));
+  }
+
+  @Test
+  public void pathEqualsInputNotNullNullOutputFalse2() {
+    Assert.assertFalse(StringUtils.pathEquals("/", null));
+  }
+
+  @Test
+  public void quoteIfStringInputNotNullOutputNotNull() {
+    Assert.assertEquals("\'/\'", StringUtils.quoteIfString("/"));
+  }
+
+  @Test
+  public void quoteInputNotNullOutputNotNull() {
+    Assert.assertEquals("\'\'\'", StringUtils.quote("\'"));
+  }
+
+  @Test
+  public void removeDuplicateStringsInput0Output0() {
+  	Assert.assertArrayEquals(new String[] {},
+			  StringUtils.removeDuplicateStrings(new String[]{}));
+  }
+
+  @Test
+  public void replaceInputNotNullNotNullNotNullOutputNotNull() {
+    Assert.assertEquals("", StringUtils.replace("", "!", ""));
+  }
+
+  @Test
+  public void sortStringArrayInput0Output0() {
+  	Assert.assertArrayEquals(new String[] {}, StringUtils.sortStringArray(new String[]{}));
+  }
+
+  @Test
+  public void sortStringArrayInput2Output2() {
+    final String[] array = {
```

Comment:
If you could find out which phase this was generated in we can look into how we can stop this from happening

Reply:
Bugged: https://diffblue.atlassian.net/browse/TG-9317